### PR TITLE
For support different python versions

### PR DESCRIPTION
**Pull request by xuehui1991**
This change could make the gradient feature selector could run in multiply python versions.

### CODE REVIEW

File: src/sdk/pynni/nni/feature_engineering/gradient_selector/fginitialize.py [hunk 1]
```diff
@@ -33,8 +33,8 @@
 # pylint: disable=E0611
 from torch.utils.data.dataloader import _DataLoaderIter, _utils
 
-import nni.feature_engineering.gradient_selector.constants as constants
-import nni.feature_engineering.gradient_selector.syssettings as syssettings
+from . import constants as constants
```
Comment (chicm-ms) [inline src/sdk/pynni/nni/feature_engineering/gradient_selector/fginitialize.py]: remove the `as constants` since they are the same?

Comment (xuehui1991) [reply src/sdk/pynni/nni/feature_engineering/gradient_selector/fginitialize.py]: Good suggestion.
